modules/index/build.gradle

```diff
@@ -32,6 +32,7 @@ dependencies {
     implementation project(':ignite-raft-api')
     implementation project(':ignite-metastorage-api')
     implementation project(':ignite-placement-driver-api')
+    implementation project(':ignite-replicator')
```

Comment:
Why is this needed?

Reply:
`IndexBuilder` needs in `ReplicaService`.